### DISCUSSION TITLE
Should I be seeing this?

### DISCUSSION THREAD

**Discussion by DanCathers** [learningeconomy/LearnCard#193] [Help] [answered]
I just created an ID, type License.  It appeared to be successful.  However, I was expecting to actually enter the relevant data from my Idaho Driver's License in to the record.  When I looked at Safari (I am doing all this on my Mac Air), there was a tab that I had not opened and it had an image that it was processing something.  Should I have seen the tab in my browser?

Dan Cathers
<img width="1280" alt="Screenshot 2022-11-24 at 11 36 54 AM" src="https://user-images.githubusercontent.com/16544299/203857224-440c839a-ff17-47e6-9dbb-bb2a4f820ede.png">

**Comment by Custard7** [comment, marked as answer]
Hey @DanCathers - that tab was opened when you initially logged-in to your LearnCard account. It usually / should close on it's own after logging in, but sometimes it stays open, and shouldn't be cause for concern. You may close it without any issues.

LearnCard uses self-custodial login by Web3Auth. This means that you - not us - fully control your account, your ID, your credentials through your own private key that we don't have access to. 

You can learn more about how it works here:

https://web3auth.io/docs/overview/how-web3auth-works
